Lib/test/test_class.py

```diff
@@ -604,12 +604,48 @@ def add(self, other):
         with self.assertRaises(TypeError):
             A() + 1
 
-    def testSetattrNonStringName(self):
```

Comment:
Seems like you removed this test.

Reply:
Yes that was on purpose as that is a duplicate of https://github.com/python/cpython/pull/25863.

Reply:
I don't think it's a duplicate, that test tests `setattr()` the function and this one tests direct calls to `type.__setattr__`.